### ISSUE TITLE
Lambert problem

### ISSUE THREAD

**Issue by joelgeorgem** [jp-praveen/Rendezvous#18]
Implement [this algorithm](http://www.ingaero.uniroma1.it/attachments/1978_2008_JGCD_Lambert.pdf) too, and compare its output with that of Curtis and Prussing.

**Comment by jp-praveen** [comment]
Yes Sir. I will implement it and let you know about the outputs.

**Comment by joelgeorgem** [comment]
Are we using [this](https://in.mathworks.com/matlabcentral/fileexchange/26348-robust-solver-for-lambert-s-orbital-boundary-value-problem)?

**Comment by jp-praveen** [comment]
Yes Sir. I downloaded that one.

**Comment by joelgeorgem** [comment]
See [this](https://zenodo.org/record/3479924#.XfDzMW5uLIU) too.

**Comment by jp-praveen** [comment]
Yes Sir. I was reading on how to use that one. I will finish it and will update you soon.

**Comment by jp-praveen** [comment]
Sir, I read the readme files and tried running the example code provided but I am getting an error when I run it. 

" Index exceeds matrix dimensions.
Error in loadlibrary>getLoadlibraryCompilerConfiguration (line 527)
Error in loadlibrary (line 263)
Error in ivLam_initializeDLL (line 72)
    loadlibrary([dllLocation 'ivLamDLL.dll'],[dllLocation 'ivLam.h'],'mfilename','ivLamDLL_matlabGenerated.m');

Error in exampleDriver_ivLamDLL (line 60)
iflag=ivLam_initializeDLL(NmaxLoad,coef_Path,dll_path);  "

Initially, I thought the problem was in the way I compiled the code, but later from MatLab forums, I found that such problems in loadlibrary may be due to compiler issues. Since mine is an older version, R2015a, I will reinstall a new version of MatLab and will try once again.

**Comment by jp-praveen** [comment]
Sir, I compiled the programs and tried with some known examples for the '0' revolution case as given in the book, but I am getting an answer that is different from the expected one. I tried using both the low-resolution and high-resolution coefficient files. I also have some doubts regarding the inputs for the multiple revolutions case.
